main.cmake

```diff
@@ -44,6 +46,18 @@ option(MENDER_ARTIFACT_GZIP_COMPRESSION "Enable GZIP compression support when do
 option(MENDER_ARTIFACT_LZMA_COMPRESSION "Enable LZMA compression support when downloading and extracting Artifacts (Default: ON)" ON)
 option(MENDER_ARTIFACT_ZSTD_COMPRESSION "Enable Zstd compression support when downloading and extracting Artifacts (Default: ON)" ON)
 
+# Coverage may fail when building tests.
+if(ENABLE_CCACHE)
+  find_program(CCACHE ccache)
+  if(CCACHE)
+    message(STATUS "Found ccache at ${CCACHE}")
+    set(CMAKE_C_COMPILER_LAUNCHER ${CCACHE})
+    set(CMAKE_CXX_COMPILER_LAUNCHER ${CCACHE})
+    set(CMAKE_LINKER_COMPILER_LAUNCHER ${CCACHE})
+  endif()
```

Comment:
Q: Should we log a warning if `ccache` is enabled and not found?

Reply:
Probably not. It's not a requirement. I could add a message at the bottom if CCache is being used.